datadog_checks_dev/changelog.d/17008.removed

```diff
@@ -0,0 +1 @@
+Remove commands that are migrated to the new `ddev` CLI
```

Comment:
wait, so we'll have 2 changelog entries for removing commands? just checking, it's probably fine.

Reply:
That's correct, with two different links. We will be able to manually edit the changelog during the release to make the changelog nicer though